### PR TITLE
qbs 1.19.0

### PR DESCRIPTION
**Pull request by ABBAPOH**
- [+] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [+] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [+] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by ABBAPOH** [comment]
Why does it say "formula deprecated"?

**Comment by SMillerDev** [comment]
Because it was deprecated by upstream: https://www.qt.io/blog/2018/10/29/deprecation-of-qbs

**Comment by dawidd6** [comment]
Doesn't seem like it is deprecated tho:

https://github.com/Homebrew/homebrew-core/pull/67552

**Comment by SMillerDev** [comment]
If you can find a notice that it's undeprecated somewhere we can remove the tag, but until then I'm going to assume that these are just security fixes and people still shouldn't use `qbs`.

**Comment by ABBAPOH** [comment]
Thanks for clarification, I now see the line in the formula (how did I miss it?). I will remove it in a separate pull-request

**Comment by ABBAPOH** [comment]
> that these are just security fixes

this is not true, new major versions include new features too

**Comment by SMillerDev** [comment]
But is the depreciation halted? Or is it more like python 2 where support is stopped but the release managers keep churning out new releases like they didn't hear anything?

**Comment by ABBAPOH** [comment]
> But is the depreciation halted?

The project is maintained and developed by the community for 2 years now. Qt Company has nothing to do with it anymore. The depreciation term was used for the build system used to build Qt itself (now it's CMake), not for the Qbs project.

**Comment by ABBAPOH** [comment]
Hm, does 11-arm64 machine use Xcode 12.5? Is there an option to use older (12.4) Xcode or skip arm tests?

**Comment by SMillerDev** [comment]
The only reason the other systems don't use xcode 12.5 is because nobody had time to update them yet. If it can't work with the newer xcode we can't ship this (yet).

File: Formula/qbs.rb
Comment (carlocab) [inline Formula/qbs.rb:7]:
```suggestion
```
You need to remove this line.

**Review by carlocab — COMMENTED** [review]
Thanks, @ABBAPOH.

For future reference, please use the preferred commit style for homebrew/core, as referenced in the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md).

PRs should have only one commit per formula modified (and only one formula modified per commit). Version bump PRs should use commit messages like
```
qbs 1.18.2
```

**Comment by ABBAPOH** [comment]
Got it, thanks!

I assume, multiple commits will be squashed during merge with the PR message? Or do I need to do a rebase + squash amending the first commit?

**Comment by carlocab** [comment]
BrewTestBot will squash as appropriate. Btw, you didn't actually remove the `deprecate!` line.

**Review by carlocab — DISMISSED** [review]

**Comment by carlocab** [comment]
Since you've already made changes additional changes here, I think you may as well squash all your commits together with the message
```
qbs 1.18.2
```

**Review by dawidd6 — DISMISSED** [review]

**Comment by dawidd6** [comment]
Sure. Slapping labels on the PR for now.

**Comment by ABBAPOH** [comment]
Can be merged now (if tests pass)

**Review by dawidd6 — APPROVED** [review]

**Pull request closed without merging**
